third_party/blink/renderer/platform/scheduler/common/throttling/budget_pool.cc

```diff
@@ -92,18 +98,10 @@ void BudgetPool::Close() {
 }
 
 void BudgetPool::UpdateStateForAllThrottlers(base::TimeTicks now) {
-  // https://linear.app/replay/issue/RUN-1045
-  recordreplay::Assert("[RUN-1045] BudgetPool::UpdateStateForAllThrottlers %d",
-                       recordreplay::PointerId(this));
-
-  std::vector<TaskQueueThrottler*> throttlers;
-  for (TaskQueueThrottler* throttler : associated_throttlers_)
-    throttlers.push_back(throttler);
-  std::sort(throttlers.begin(), throttlers.end(), recordreplay::CompareByPointerId());
-  for (TaskQueueThrottler* throttler : throttlers) {
-    // https://linear.app/replay/issue/RUN-1045
-    recordreplay::Assert("[RUN-1045] BudgetPool::UpdateStateForAllThrottlers #1 %d",
-                         recordreplay::PointerId(throttler));
+  for (TaskQueueThrottler* throttler : associated_throttlers_) {
```

Comment:
Undo most of our changes:
* Only keep the (now updated) Assert.
* Order is assured by the change to `associated_throttlers_`.